nibabel/tests/test_info.py

```diff
@@ -0,0 +1,47 @@
+""" Testing info module
+"""
+
+import nibabel as nib
+from nibabel import info
+from nibabel.info import cmp_pkg_version
+
+from nose.tools import (assert_raises, assert_equal)
+
+
+def test_version():
+    # Test info version is the same as our own version
+    assert_equal(info.__version__, nib.__version__)
+
+
+def test_cmp_pkg_version():
+    # Test version comparator
+    assert_equal(cmp_pkg_version(info.__version__), 0)
+    assert_equal(cmp_pkg_version('0.0'), -1)
+    assert_equal(cmp_pkg_version('1000.1000.1'), 1)
+    assert_equal(cmp_pkg_version(info.__version__, info.__version__), 0)
+    for test_ver, pkg_ver, exp_out in (('1.0', '1.0', 0),
+                                       ('1.0.0', '1.0', 0),
+                                       ('1.0', '1.0.0', 0),
+                                       ('1.1', '1.1', 0),
+                                       ('1.2', '1.1', 1),
+                                       ('1.1', '1.2', -1),
+                                       ('1.1.1', '1.1.1', 0),
+                                       ('1.1.2', '1.1.1', 1),
+                                       ('1.1.1', '1.1.2', -1),
+                                       ('1.1', '1.1dev', 1),
+                                       ('1.1dev', '1.1', -1),
+                                       ('1.2.1', '1.2.1rc1', 1),
+                                       ('1.2.1rc1', '1.2.1', -1),
+                                       ('1.2.1rc1', '1.2.1rc', 1),
+                                       ('1.2.1rc', '1.2.1rc1', -1),
+                                       ('1.2.1rc1', '1.2.1rc', 1),
+                                       ('1.2.1rc', '1.2.1rc1', -1),
+                                       ('1.2.1b', '1.2.1a', 1),
+                                       ('1.2.1b', '1.2.1a', 1),
```

Comment:
With this comparator, I'm assuming that any letter suffix to a version, makes the version earlier than the same version without the letter suffix.  So '-rc1' 'a' 'b' 'dev' all make the version earlier.  In particular, I'd take 'a' to mean 'alpha' and 'b' to mean 'beta'.  Of course someone could try and use 'post' as a suffix, and expect that to be later, but we haven't done that yet, and so I was hoping to ignore that.